index.js

```diff
@@ -0,0 +1,21 @@
+
+const mongoose = require('mongoose');
+const Note = require('./lib/models/Notes.js'); 
+const { Input } = require('./lib/Input');
+let parsedArray = new Input(process.argv); 
+let response; 
+console.log(parsedArray.isValid()) 
+
+if(parsedArray.isValid()) {
+    response = Note.execute(parsedArray); 
+} else {
+    console.log('try again please')
+}
+
+const action = parse(process.argv);
+const isValid = valid(action); 
+
+// mongoose.connect('mongodb://localhost:27017/notelist', {
```

Comment:
You can structure your index.js file like this:

const mongoose = require('mongoose');
const Input = require('./lib/Input')
const Note = require('./lib/models/Note');
mongoose.connect('mongodb://localhost:27017/notes', {
  useNewUrlParser: true,
  useUnifiedTopology: true
});
const input = new Input(process.argv);
if(input.valid()) {
  Note.execute(input)
    .then(() => mongoose.connection.close());
} else {
  console.log('Something went wrong');
}